composer.json

```diff
@@ -4,19 +4,27 @@
 	"license"    : "GPL-2.0-or-later",
 	"version"    : "0.4.2",
 	"require"    : {
-		"php"                 : ">=7.1",
-		"phpunit/phpunit"     : ">=7.0",
-		"mockery/mockery"     : "^1.0",
+		"php"                 : ">=7.1 < 9.0",
+		"phpunit/phpunit"     : "^9.5.24",
+		"mockery/mockery"     : "^1.5",
 		"antecedent/patchwork": "^2.1"
 	},
 	"require-dev": {
 		"behat/behat"         : "^3.0",
 		"sebastian/comparator": ">=1.2.3",
-		"php-coveralls/php-coveralls": "^2.1"
+		"php-coveralls/php-coveralls": "^2.1",
+		"sempro/phpunit-pretty-print": "^1.4"
 	},
 	"autoload"   : {
-		"psr-4"   : {"WP_Mock\\": "./php/WP_Mock"},
-		"classmap": ["php/WP_Mock.php"]
+		"psr-4"   : {
+		  "WP_Mock\\": "./php/WP_Mock"
+		},
+		"classmap": [
+		  "php/WP_Mock.php"
+		]
+	},
+	"config": {
+		"platform-check": false
```

Comment:
Why was this added?

Reply:
sorry what I really meant was this: https://github.com/10up/wp_mock/pull/167/commits/82d4ec65cf034a8ef6a628bf609619fb3cb0a023
thanks for reminding me